plugins/linux-v4l2/v4l2-output.c

```diff
@@ -50,10 +50,24 @@ static bool loopback_module_loaded()
 	return loaded;
 }
 
+bool loopback_module_available()
+{
+	if (loopback_module_loaded()) {
+		return true;
+	}
+
+	if (system("PATH=\"$PATH:/sbin\" modinfo v4l2loopback >/dev/null 2>&1") ==
```

Comment:
many linux distros are moving binaries to /usr... so you want /usr/sbin also in the path as /sbin might go away at some point